### PR TITLE
Fix error with Telemetry Client Filter

### PR DESCRIPTION
**Pull request by Yas-Franco**

### CODE REVIEW

**Comment by Yas-Franco** [comment]
#build

**Comment by Yas-Franco** [comment]
#build

File: dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry/internal/rest/TelemetryClientFilter.java [hunk 1]
```diff
@@ -27,11 +27,15 @@
...
 
+import jakarta.enterprise.inject.spi.CDI;
+
 import io.opentelemetry.api.OpenTelemetry;
 import io.opentelemetry.context.Context;
 import io.opentelemetry.context.Scope;
 import io.opentelemetry.context.propagation.TextMapSetter;
 import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+
+import io.opentelemetry.api.trace.Tracer;
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry/internal/rest/TelemetryClientFilter.java:38]: Is this an unused import?

File: dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry/internal/rest/TelemetryClientFilter.java [hunk 2]
```diff
@@ -27,11 +27,15 @@
 import jakarta.ws.rs.core.UriBuilder;
 import jakarta.ws.rs.ext.Provider;
 
+import jakarta.enterprise.inject.spi.CDI;
```
Comment (fmhwong) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry/internal/rest/TelemetryClientFilter.java:30]: Is this an unused import?

**Review by fmhwong — APPROVED** [review]
LGTM.  The unused import statements can be fixed in a future PR.

**Comment by Yas-Franco** [comment]
#libby

**Pull request merged**